### DISCUSSION TITLE
Account Restoration Request

### DISCUSSION THREAD

**Discussion by yinkai1205** [community/community#63216] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Dear GitHub Support Team,

I hope you are doing well. I am writing to request the restoration of my GitHub account, which has been recently suspended. I apologize if there were any violations from my end, and I am committed to complying with GitHub’s terms of service and community guidelines going forward.

I kindly ask for your assistance in providing specific details regarding the reason for the suspension and any necessary steps I need to take to resolve the issue. If there are any documents or information required for identity verification, please let me know, and I will promptly provide them.

GitHub has been an essential platform for my development projects and collaboration with the open-source community. The suspension has impacted my ability to contribute and learn from fellow developers. I would greatly appreciate your prompt attention to this matter.

Thank you for your understanding and support.

Sincerely,
Kevin
[REDACTED DUE TO PII]

**Comment by queenofcorgis** [comment, marked as answer]
Hi @yinkai1205,

The community is unable to assist with nor escalate your situation, but the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Comment by Aliyan-Ansari** [comment]
Hello @queenofcorgis 

I hope this message finds you well. I am reaching out with a humble request regarding a recent GitHub discussion that I was involved in. The discussion in question is located at the following link: [GitHub Discussion Link](https://github.com/orgs/community/discussions/61353#discussioncomment-6499272).

I understand and respect GitHub's policies regarding acceptable use and coordinated activity. I want to express my sincere apologies if my previous contributions inadvertently raised concerns in this regard. My intentions were solely focused on seeking assistance and participating in meaningful discussions within the community.

In light of the situation, I kindly request your assistance in deleting the mentioned thread, rather than locking it. I believe that removing the thread entirely would help rectify any misunderstanding and ensure that there are no future repercussions. I am committed to upholding the integrity of the GitHub community and will ensure a more careful approach in my interactions moving forward.

Once again, I apologize for any inconvenience or confusion my previous engagement may have caused. Your understanding and support in this matter are greatly appreciated.

Thank you for your time and attention. I look forward to your positive response.

**Reply by yinkai1205** [reply]
However, I am unable to use my account. What should I do to recover it? Thank you very much! Account: **Kevin1205888**  [REDACTED DUE TO PII].

**Reply by queenofcorgis** [reply]
@yinkai1205 can you use this account to open a ticket? Just be sure to include your username/email of the account in question.

**Reply by Aliyan-Ansari** [reply]
@queenofcorgis, I kindly request your assistance in reviewing my current inquiry. I find myself uncertain about the appropriate course of action, including whom to approach and the appropriate channels for communication. Your guidance would be greatly appreciated. Thank you.

**Reply by queenofcorgis** [reply]
@Aliyan-Ansari There is no action needed on your part. If you'd like to review GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) and [Code of Conduct](https://github.com/community/community/blob/9a181a9c8c825c5d2c59b2749c0d7114bfd4d38a/CODE_OF_CONDUCT.md) - I've linked them here.

**Comment by saumyasunder** [comment]
Dear GitHub team,
I have the following account SaumyaSunder and looks like there is a bot that has hacked my account with a repo https://github.com/saumyasunder/github-slideshow

I am unable to delete this account nor am I able to reclaim this. Could you please help me with this issue?

Thanks
Saumya Sunder
